uui-components/src/table/DataTableRow.tsx

```diff
@@ -36,8 +36,8 @@ export class DataTableRow<TItem, TId> extends Component<DataTableRowProps<TItem,
                 rawProps={ {
                     ...params.eventHandlers,
                     role: 'row',
-                    'aria-expanded': this.props.isFolded == undefined ? undefined : this.props.isFolded,
-                    ...(this.props.isSelectable && { 'aria-selected': this.props.isSelected } )
+                    'aria-expanded': !this.props.isFolded ?? undefined,
```

Comment:
It seems this code is not fully correct:
* `isFolded === undefined` → `aria-expanded: true`
* `isFolded === false` → `aria-expanded: true`
* `isFolded === true` → `aria-expanded: undefined`

I suppose the initial solution was correct, except there should be `!` symbol added before `this.props.isFolded`.

CC @AlekseyManetov